README.md

```diff
@@ -1 +1,226 @@
-# Hello-Microverse
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup) & [Install](#install)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 Hello-Microverse
+
+> Description 
+
+The goal of this project is to set up linters in your HTML and CSS project to improve the quality of your code and help catch errors and potential issues. Linters are tools that let you check your code to make sure it follows coding best practices. By using this project, you will be able to improve the quality of your code and save time by avoiding common mistakes.
+
+
+
+## 🛠 Built With 
+
+This project was carried out using the following tools:
+
+Visual Studio Code - an open source and free code editor
+
+Terminal - a terminal emulator for running commands
+
+GitHub - a software development platform based on Git
+
+ESLint - a JavaScript linter to detect coding errors and potential issues
+
+Stylelint - a CSS linter to detect syntax errors and formatting issues in CSS files
+
+
+
+
+### Tech Stack
+
+This project was carried out using the following technologies:
+
+HTML - the standard markup language for creating web pages
+CSS - a style language used to define the presentation of web pages
+GitHub - a software development platform based on Git
+
+
```

Comment:
- [ ] Great job making the changes requested by the previous reviewer. The README file you added can be further edited to make it even more professional. The section **Built With**, should be formatted in a similar fashion that it was in the template. This means you follow an orderly format like so 👇🏾 


```html
## 🛠 Built With <a name="built-with"></a>
 > If you want you can add some text as you did, but make sure it is easy to read. preferably in bullet points
### Tech Stack <a name="tech-stack"></a>
<details>
  <summary>Client</summary>
  <ul>
    <li><a href="">HTML</a></li>
    <li><a href="">CSS</a></li>
  </ul>
</details>

```

Yu can see below the way this section is in your README. Not easy to read.

![image](https://user-images.githubusercontent.com/84156714/219040575-859693d3-ce3a-4dc8-8351-00cf83d72f41.png)